mon-pix/tests/integration/components/footer_test.js

```diff
@@ -2,33 +2,31 @@ import Service from '@ember/service';
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
 import setupIntlRenderingTest from '../../helpers/setup-intl-rendering';
-import { find, findAll, render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
-
-import { contains } from '../../helpers/contains';
+import { render } from '@1024pix/ember-testing-library';
 
 describe('Integration | Component | Footer', function () {
   setupIntlRenderingTest();
 
   it('should display the Pix logo', async function () {
     // when
-    await render(hbs`<Footer />}`);
+    const screen = await render(hbs`<Footer />}`);
 
     // then
-    expect(find('.pix-logo__image')).to.exist;
+    expect(screen.getByAltText(this.intl.t('navigation.homepage'))).to.exist;
   });
 
   it('should display the navigation menu with expected elements', async function () {
     // when
-    await render(hbs`<Footer />}`);
+    const screen = await render(hbs`<Footer />}`);
 
     // then
-    expect(findAll('.footer-navigation__item')).to.have.lengthOf(5);
-    expect(contains(this.intl.t('navigation.footer.a11y'))).to.exist;
-    expect(contains(this.intl.t('navigation.footer.data-protection-policy'))).to.exist;
-    expect(contains(this.intl.t('navigation.footer.eula'))).to.exist;
-    expect(contains(this.intl.t('navigation.footer.help-center'))).to.exist;
-    expect(contains(this.intl.t('navigation.footer.sitemap'))).to.exist;
+    expect(screen.getByText(this.intl.t('navigation.footer.a11y'))).to.exist;
+    expect(screen.getByText(this.intl.t('navigation.footer.data-protection-policy'))).to.exist;
```

Comment:
Suggestion : Top de nettoyer les tests pour Testing Library. 
Je peut juste te conseiller de privilégier l'utilisation de la requête getByRole et si tu ne peut pas passer par celle-ci alors getByText conviendra très bien.

Ici s'il s'agit de s'assurer de la présence de lien, on peut faire ça : 

assert.dom(screen.getByRole('link', { name: this.intl.t('navigation.footer.data-protection-policy' })).exists();

Reply:
En plus je l'ai écris comme ça dans le test que j'ai ajouté 🤦‍♀

Du coup yes ça fonctionne très bien avec ```getByRole``` aussi ! J'ai mis à jour le commit